yarn.lock

```diff
@@ -842,6 +842,13 @@ aws4@^1.8.0:
   resolved "https://registry.npmjs.org/aws4/-/aws4-1.10.0.tgz"
   integrity sha512-3YDiu347mtVtjpyV3u5kVqQLP242c06zwDOgpeRnybmXlYYsLbtTrUBUm8i8srONt+FWobl5aibnU1030PeeuA==
 
+axios@^0.21.1:
+  version "0.21.1"
+  resolved "https://nexus.uploan.ph/repository/npm-package/axios/-/axios-0.21.1.tgz#22563481962f4d6bde9a76d516ef0e5d3c09b2b8"
```

Comment:
It's using the Nexus proxy to download public packages. As a result, this project would need Nexus access. I don't think this is intentional since the rest of the existing packages were pulled from `https://registry.npmjs.org`. This may have been caused by an `npm` or `yarn` configuration.